README.md

```diff
@@ -209,7 +209,8 @@ implements the following extensions:
     to use 2 backslashes it the end of the line.\\
 
 *   **Includes**, support including files with `{{filename}}` syntax. This is only
-    done when include is started at the beginning of a line.
+    done when include is started at the beginning of a line. The filename is
+    specified with forward slashes `/`.
```

Comment:
I would word this as 'The path separator is for the forward slash `/`. Even on non-Unix environments'.'